packages/client/components/JiraScopingSearchResults.tsx

```diff
@@ -36,6 +83,7 @@ const JiraScopingSearchResults = (props: Props) => {
   const usedServiceTaskIds = useGetUsedServiceTaskIds(estimatePhase)
   const handleAddIssueClick = () => setIsEditing(true)
 
+  // TODO: after add Suspense fallback in Root, should be able to remove this?
   // even though it's a little herky jerky, we need to give the user feedback that a search is pending
   // TODO fix flicker after viewer is present but edges isn't set
   if (!edges) {
```

Comment:
`<MockScopingList />` here is a duplicate with the `fallback` of Suspense at the upper level

Reply:
okay, so I also remove the related comments here.